js/views/userPageVw.js

```diff
@@ -1444,6 +1467,70 @@ module.exports = baseVw.extend({
 
   },
 
+  addModeratorClick: function(e){
+    var $targ = $(e.target).closest('.js-addmoderator'),
+        self = this,
+        modList = {};
+
+    $targ.addClass('loading');
+
+    modList.moderators = this.model.get('user').moderators;
```

Comment:
The moderators you're getting here is an array of objects. It needs to be an array of guids.

If you look in userMd.js, it already pulls the guids from the moderator objects and stores them in moderator_guids.

If you change this to get('user').moderator_guids it should work.

As-is, this will erase all of your moderators except the one you're adding.